C/services/filter-plugin-interfaces/python/python_plugin_interface.cpp

```diff
@@ -161,32 +170,44 @@ static void filter_plugin_reconfigure_fn(PLUGIN_HANDLE handle,
  */
 void filter_plugin_ingest_fn(PLUGIN_HANDLE handle, READINGSET *data)
 {
-	if (!pythonModules)
+        if (!handle)
 	{
-		Logger::getLogger()->error("pythonModules map is NULL "
-					   "in filter_plugin_ingest_fn plugin '%s'",
-					   gPluginName.c_str());
+		Logger::getLogger()->fatal("plugin_handle: filter_plugin_ingest_fn(): "
+					   "handle is NULL");
 		return;
 	}
 
-	auto it = pythonModules->find(gPluginName);
-	if (it == pythonModules->end() ||
+	if (!pythonHandles)
+	{
+		// Plugin name can not be logged here
+		Logger::getLogger()->error("pythonHandles map is NULL "
```

Comment:
Can we say something more meaningful to an end user here? Or is this a truly unexpected event?

Reply:
Unexpected event